### ISSUE TITLE
View assets available to deposit

### ISSUE THREAD

**Issue by jahabeebs** [open-dollar/od-app#239]
Part of TVL project https://github.com/open-dollar/od-app/issues/234
Depends on #238 

## Description
- Users should be able to view assets available for deposit, specifically wstETH and rETH

![image](https://github.com/open-dollar/od-app/assets/35622595/4c5a92c7-ba06-49e8-85e9-fd048b2f747e)

**Comment by pi0neerpat** [comment]
I created the route, and rough buttons for navigation. You'll need to coordinate with Jacob on where to pull this data from. Or if you complete your side first, then he can wire the data up in another issue
